test/repl.jl

```diff
@@ -159,8 +159,8 @@ end
 function AddCustomMode(repl)
     # Custom REPL mode tests
     foobar_mode = LineEdit.Prompt("Test";
-        prompt_prefix="\e[38;5;166m",
-        prompt_suffix=Base.text_colors[:white],
+        prompt_prefix="\e[38;5;166mΠ",
```

Comment:
for seeing if anything broke if random unicode characters were inserted (although it looks like I forgot to insert them into the prompt itself as I had also intended :/ )